google-cloud-bigquery/pom.xml

```diff
@@ -126,6 +126,11 @@
       <artifactId>proto-google-cloud-datacatalog-v1</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Can we use `gson` (already included) instead? `org.json` is being banned by the Apache foundation.

Reply:
Great idea! Did just that